src/HomeWork/Sum/SumMain.java

```diff
@@ -1,33 +1,34 @@
 package HomeWork.Sum;
 
-import HomeWork.NumbRevers.ReversMethod;
-
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class SumMain {
     public static void main(String[] args) {
-        boolean miss = false;
-        System.out.print("Enter the numbers that you want to sum: ");
-        int sum =0;
-        int numbers = 0;
-
+        boolean isExit;
         do {
-            Scanner sc = new Scanner(System.in);
-            numbers = sc.nextInt();
-            sum+= numbers;
-
-            }while (numbers!=0);
             try {
-                System.out.println("Sum" + sum);
-
-
+                System.out.println("Enter the required size of the array : ");
+                Scanner s = new Scanner(System.in);
+                int size = s.nextInt();
+                int myArray[] = new int[size];
+                int sum = 0;
+                System.out.println("Enter the elements of the array one by one ");
+
+                for (int i = 0; i < size; i++) {
+                    myArray[i] = s.nextInt();
+                    sum = sum + myArray[i];
+                }
+                System.out.println("Elements of the array are: " + Arrays.toString(myArray));
+                System.out.println("Sum of the elements of the array :" + sum);
 
             } catch (Exception mistake) {
-                miss = true;
+
                 System.out.println("Wrong type" + mistake.getMessage());
-            }
+                System.out.println("Please try again");
 
-        while (miss);
+            }
+        } while (isExit = true);
```

Comment:
it will be enough to write just 'while (isExit)'

Reply:
Wont to be initilization...